lib/runtime/src/distributed.rs

```diff
@@ -104,14 +113,17 @@ impl DistributedRuntime {
         Ok(distributed_runtime)
     }
 
-    pub async fn from_settings(runtime: Runtime) -> Result<Self> {
-        let config = DistributedConfig::from_settings(false);
+    pub async fn from_settings(
```

Comment:
from_settings means from settings, not from settings and...

we need to use a builder pattern or a different from_